WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -417,15 +415,26 @@ private void updateEmptyView(EmptyViewMessageType emptyViewMessageType) {
                 return;
         }
 
-        mEmptyViewTitle.setText(getText(stringId));
-        mEmptyViewImage.setVisibility(emptyViewMessageType == EmptyViewMessageType.NO_CONTENT ? View.VISIBLE
-                                              : View.GONE);
-        mEmptyView.setVisibility(isPostAdapterEmpty() ? View.VISIBLE : View.GONE);
+        boolean hasNoContent = emptyViewMessageType == EmptyViewMessageType.NO_CONTENT;
+        mActionableEmptyView.setImageResource(mIsPage ? R.drawable.img_illustration_pages_104dp
+                : R.drawable.img_illustration_posts_75dp);
+        mActionableEmptyView.setImageVisibility(hasNoContent);
+        mActionableEmptyView.setTitleText(getText(stringId));
+        mActionableEmptyView.setButtonText(getString(mIsPage ? R.string.pages_empty_list_button
+                : R.string.posts_empty_list_button));
+        mActionableEmptyView.setButtonVisibility(hasNoContent);
+        mActionableEmptyView.setButtonClickListener(new OnClickListener() {
+            @Override public void onClick(View view) {
+                ActivityLauncher.addNewPostOrPageForResult(getActivity(), mSite, mIsPage, false);
+            }
+        });
+        mActionableEmptyView.setVisibility(isPostAdapterEmpty() ? View.VISIBLE : View.GONE);
+        mSwipeRefreshLayout.setEnabled(!isPostAdapterEmpty());
```

Comment:
Correct me if I'm wrong, but If I'm reading the code correctly, this will disable the PTR when we load the posts (post adapter will still be empty during loading) and will not enable it later (we won't call `updateEmptyView()` unless there is some sort of error or no content).

Reply:
The `mSwipeRefreshLayout.setEnabled(!isPostAdapterEmpty());` statement was meant to just disable/enable `mSwipeRefreshLayout` when `mActionableEmptyView` is visible/invisible.

> this will disable the PTR when we load the posts (post adapter will still be empty during loading)

I think this should be expected behavior.

> will not enable it later... unless there is some sort of error or no content

This is not expected behavior.  I added enabling swipe-to-refresh when the empty view is hidden in 2abcb18.